package.json

```diff
@@ -118,6 +118,7 @@
     "simplemde": "https://github.com/kevinansfield/simplemde-markdown-editor.git#ghost",
     "testem": "2.0.0",
     "top-gh-contribs": "2.0.4",
+    "validator": "7.2.0",
```

Comment:
validator's latest version is technically 9.x, however there was a breaking change in 8.x that changed how the isURL function worked with localhost urls. Probably could update it to 9.x, but that should be a separate pr IMO, as the usages of isURL will need to be evaulated. (For what it's worth, Ghost-CLI is stuck on validator 7.2 for the same reason)

Reply:
👍